### PR TITLE
cleanup the initialize_surface in Cases.jl

### PR DESCRIPTION
**Pull request by yairchn**

### CODE REVIEW

**Review by charleskawczynski — COMMENTED** [review]
The code changes look fine to me. It would be helpful for someone to review the equations. It looks like Rico is noticeably affected by these changes, are these changes okay?

**Review by ilopezgp — APPROVED** [review]
LGTM. zero initial fluxes in TRMM is consistent with the PyCLES implementation of the forcing.

**Comment by yairchn** [comment]
@charleskawczynski @trontrytel are you ok with this PR ?

File: driver/Cases.jl [hunk 1]
```diff
@@ -775,8 +779,8 @@ function initialize_surface(self::CasesBase{TRMM_LBA}, grid::Grid, state, param_
     theta_surface = (273.15 + 23)
     ts = TC.thermo_state_pθq(param_set, p0_f_surf, theta_surface, self.Sur.qsurface)
     self.Sur.Tsurface = TD.air_temperature(ts)
-    self.Sur.lhf = 5.2e-5 * ρ0_f_surf * TD.latent_heat_vapor(ts)
-    self.Sur.shf = 8.0e-3 * TD.cp_m(ts) * ρ0_f_surf
+    self.Sur.lhf = 0.0
```
Comment (trontrytel) [inline driver/Cases.jl:782]: Were we using a wrong setup for trmm before? Should they be zero?

**Review by trontrytel — APPROVED** [review]
I like the temperature constructors better for sure. Left a comment about TRMM surface fluxes. lgtm

**Review by charleskawczynski — APPROVED** [review]

File: driver/Cases.jl (hunk 1 above)
Comment (yairchn) [reply driver/Cases.jl:782]: yes the surface flux starts at zero an increases over time

**Comment by yairchn** [comment]
Bors r+

**Pull request merged**